src/main/java/com/exercise/sfe/entity/GiftCertificate.java

```diff
@@ -0,0 +1,20 @@
+package com.exercise.sfe.entity;
+
+import lombok.Data;
+import lombok.ToString;
+
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Data
+@ToString(callSuper = true) // is it ok or I should do it in another way?
```

Comment:
If you need it why not, but never ask questions in a such way. Better ask them directly instead of comments here